clang/lib/Tooling/DependencyScanning/ModuleDepCollector.cpp

```diff
@@ -835,6 +850,13 @@ void ModuleDepCollectorPP::addAllSubmoduleDeps(
   });
 }
 
+void ModuleDepCollectorPP::addClangModule(const Module *M, const ModuleID ID,
```

Comment:
How about something like "addOneModuleDep"?  If find it hard to guess from the name how this is different from e.g. `addModuleDep` which also has a similar signature.